apps/application-system/form/infra/application-system-form.ts

```diff
@@ -13,7 +11,7 @@ export const serviceSetup = (services: {
         dev: '',
         prod: '',
         staging: '',
-        local: ref((h) => `http://${h.svc(services.api)}`),
+        local: '',
```

Comment:
this `SI_PUBLIC_GRAPHQL_PATH` now looks rather redundant it is setting all envs to empty string?